docs/04-reference/wingsdk-spec.md

```diff
@@ -98,7 +98,7 @@ For example, a `cloud.Bucket` can emit events whenever an object is uploaded, an
 ```ts
 // wing
 let bucket = new cloud.Bucket();
-bucket.onCreate(inflight (file: str) => {
+bucket.onCreate(inflight (file: string) => {
```

Comment:
`str` is the name of the type in Wing, so let's keep it that way throughout the doc
```suggestion
bucket.onCreate(inflight (file: str) => {
```

Reply:
The reason why I changed `str` to `string` is actually because there is currently a mix of both as you can see [HERE](https://github.com/winglang/wing/blob/main/docs/04-reference/wingsdk-spec.md?plain=1#L583).

So you confirm that `str` should be preferred over `string` throughout the SDK Spec?

Reply:
> So you confirm that `str` should be preferred over `string` throughout the SDK Spec?

Yap